tasks/main.yml

```diff
@@ -1,160 +1,7 @@
-    owner: root
-    group: root
-    mode: "0644"
-  notify:
-    - restart apache
-    - restart php-fpm
-
-- name: (WordPress) Create httpd virtual host
-  template:
-    src: etc/httpd/conf.d/site.conf.j2
-    dest: "{{ apache_config_site_path }}/{{ site_domain }}.conf"
-  notify:
-    - restart apache
-
-#
-# PHP
-#
-
-- name: (WordPress) Place site-specific php-fpm config
-  template:
-    src: etc/php-fpm.d/site.conf.j2
-    dest: "{{ php_fpm_config_pool_path }}/{{ site_domain }}.conf"
-    owner: root
-    group: root
-    mode: "0644"
-  notify:
-    - restart php-fpm
-
-- name: (WordPress) Remove default www pool
-  file:
-    path: "{{ item }}"
-    state: absent
-  with_items:
-    - /etc/php/7.0/fpm/pool.d/www.conf
-    - /etc/php-fpm.d/www.conf
-  notify:
-    - restart php-fpm
-
-#
-# WordPress
-#
-
-- name: (WordPress) Check for existing WordPress installation
-  stat:
-    path: "{{ wp_system_path }}/wp-config.php"
-  changed_when: not wordpress_installed.stat.exists
-  ignore_errors: true
-  register: wordpress_installed
-
-- name: (WordPress) Install WordPress
-  include: "wordpress.yml"
-  when: wordpress_installed.changed
-
-- name: (WordPress) Get WordPress Site Url
-  command: >-
-    wp option get siteurl
-  args:
-    chdir: "{{ wp_system_path }}"
-  become: true
-  become_user: "{{ system_user }}"
-  register: siteurl
-  changed_when: false
-
-- name: (WordPress) Replace all site urls in database if changed
-  command: >-
-    wp search-replace "{{ siteurl.stdout }}" "{{ wp_protocol }}{{ site_domain }}" --skip-columns=guid
-  args:
-    chdir: "{{ wp_system_path }}"
-  become: true
-  become_user: "{{ system_user }}"
-  when: siteurl.stdout is defined and siteurl.stdout != wp_protocol + site_domain
-  vars:
-    wp_protocol: >-
-      {{ "https://"
-      if use_letsencrypt is defined
-      and use_letsencrypt
-      else "http://" }}
-
-- name: (WordPress) Ensure WordPress siteurl and homeurl are correct
-  command: >-
-    wp option set {{ item }} "{{ wp_protocol }}{{ site_domain }}"
-  args:
-    chdir: "{{ wp_system_path }}"
-  become: true
-  become_user: "{{ system_user }}"
-  changed_when: false
-  vars:
-    wp_protocol: >-
-      {{ "https://"
-      if use_letsencrypt is defined
-      and use_letsencrypt
-      else "http://" }}
-  with_items:
-    - siteurl
-    - home
-
-- name: (WordPress) Ensure content matches site protocol scheme
-  command: >-
-    wp search-replace
-
-    "{{ 'http://'
-    if use_letsencrypt is defined
-    and use_letsencrypt
-    else 'https://' }}"
-
-    "{{ 'https://'
-    if use_letsencrypt is defined
-    and use_letsencrypt
-    else 'http://' }}"
-  args:
-    chdir: "{{ wp_system_path }}"
-  become: true
-  become_user: "{{ system_user }}"
-  changed_when: false
-
-#
-# UltraStack
-#
-
-- name: (UltraStack) Include UltraStack configuration
+- name: (UltraStack) Deploy UltraStack configuration if enabled
   include: "ultrastack.yml"
   when:
     - use_ultrastack is defined
```

Comment:
This should be changed to `use_ultrastack is true` to avoid situations where the variable is set to false.

Reply:
Hey, thanks for checking on this. The full block is:
```yaml
- name: (UltraStack) Deploy UltraStack configuration if enabled
  include: "ultrastack.yml"
  when:
    - use_ultrastack is defined
    - use_ultrastack
```
`use_ultrastack` is implicitly true or false and acted on appropriately in this conditional AND list.
Hope this clarifies things.